pages/index.js

```diff
@@ -19,27 +20,45 @@ export async function getServerSideProps() {
   const URL =
     'https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/ids/mapping.json';
 
+  const membersArr = [];
+  let memberDetailsUrl = '';
+  let resp1 = '';
+  let jsonObj = '';
+
   try {
     const res = await fetch(URL);
     if (res.status !== 200) {
       throw new Error(
         'There was some issues fetching the members, Please try again after some time'
       );
     }
-    const data = await res.json();
-    return { props: { data } };
+    let data = await res.json();
+
+    for (const rdsid in data) {
+      memberDetailsUrl = getDataURL(rdsid);
+      resp1 = await fetch(memberDetailsUrl);
+      if (resp1.status == 200) {
+        jsonObj = await resp1.json();
+        membersArr.push({
+          ...jsonObj,
+          img_url: getImgURL(rdsid)
+        });
+      }
+    }
+
+    return { props: { membersArr } };
```

Comment:
Can be parallelized

Reply:
I was randomly checking this MR and am confused with the word parallelized. Can some explain me what is paralleliztion?